### ISSUE TITLE
如何使用tensorboard记录训练损失

### ISSUE THREAD

**Issue by possible1402** [bojone/bert4keras#140]
我使用example中的task_sequence_labeling_ner_crf.py训练命名实体识别模型，请问如何使用tensorboard记录训练过程中的损失呢？
`model.compile(
    loss=CRF.sparse_loss,
    optimizer=Adam(learing_rate),
    metrics=[CRF.sparse_accuracy]
)`

```
    model.fit_generator(
        train_generator.forfit(),
        steps_per_epoch=len(train_generator),
        epochs=epochs,
        callbacks=[evaluator]
    )
```
是在model.compile中设置吗，还是在model.fit_generator中呢？

**Comment by bojone** [comment]
https://keras.io/callbacks/

**Comment by possible1402** [comment]
谢谢
